contracts/Platform.sol

```diff
@@ -21,12 +24,97 @@ contract Platform {
         eventContract = eventAddr;
     }
 
+    struct bidInfo {
+        uint256 quantity;
+        uint256 pricePerTicket;
+        uint256 tokenPerTicket;
+        uint256 firstIndexForEventBiddings; // For ease of updating bid
+    }
+
+    mapping(uint256 => uint256) public eventTopBid; // eventId => topBid
+    mapping(uint256 => mapping(uint256 => address[])) public eventBiddings; // eventId => (tokenBid => addressArray) 
```

Comment:
Understand that tokenBid is used as a key & users bidding that amount will be inserted as the value. Will it be an issue if user were to update their tokenBid with this data structure?

Reply:
I am not sure if this is the best way to implement it but I think it should work? If there's a better suggestion do let me know